sources/packages/web/src/types/contracts/supporting-user/SupportingUserContracts.ts

```diff
@@ -67,3 +67,28 @@ export interface ApplicationSupportingUsersDTO {
   supportingUserId: number;
   supportingUserType: SupportingUserType;
 }
+
+export interface AddressInfo {
+  addressLine1: string;
+  addressLine2?: string;
+  province: string;
+  country: string;
+  city: string;
+  postalCode: string;
+}
+export interface ContactInfos {
+  addresses: Array<AddressInfo>;
+  phone: string;
+}
+
+export interface SupportingUserFormData {
+  formName: string;
+  supportingData: any;
+  contactInfo?: ContactInfos;
```

Comment:
Are you going to be updating this nullable also?

Reply:
this wont be nullable